src/overlays/effects/ovl_Effect_Ss_Fhg_Flash/z_eff_ss_fhg_flash.c

```diff
@@ -25,15 +25,15 @@ EffectSsInit Effect_Ss_Fhg_Flash_InitVars = {
     EffectSsFhgFlash_Init,
 };
 
-static UNK_TYPE D_809A5178[258];
-static Gfx D_809A5100[15];
+// Should eventually come from assets/overlays/ovl_Effect_Ss_Fhg_Flash/ovl_Effect_Ss_Fhg_Flash.h
```

Comment:
Because ZAPD doesn't write anything to the .h for whatever reason, this is `assets/overlays/ovl_Effect_Ss_Fhg_Flash/ovl_Effect_Ss_Fhg_Flash.h` :
```c
#ifndef OVL_EFFECT_SS_FHG_FLASH_H
#define OVL_EFFECT_SS_FHG_FLASH_H 1


#endif
```

Reply:
I think it is because everything is extracted as `static` so it doesn't get put in `.h` file since they are meant to be internal only.

Reply:
I would argue the .h itself is internal only,
but I guess it'd be weird to have either externs for static symbols, or forward static declarations which would amount to bss if there's no actual declaration later on 🤷

Should I change this comment somehow if we shouldn't assume ZAPD will ever change this in some way?

Reply:
eventually tm tm these kinda of files wont be extracted anyway (though the assets will), so id say its a temporary situation regardless. but the comment is fine imo

Reply:
Perhaps just add a todo, with a comment to investigate ZAPD forward declaring static variables. Also note you can't just extern them in the `.h` file, the compiler yells at you.

Reply:
I added a TODO 403201da867853c889480944eeeca640fb631fdb